trade-finance-manager-api/src/graphql/helpers/percentageToNumber.helper.js

```diff
@@ -0,0 +1,10 @@
+// converts string to number percentage - eg '70%' to 70
+const percentageToNumber = (percentage) => {
+  if (typeof percentage === 'string' && percentage) {
```

Comment:
can we revert the order of these 2? i.e.
```
if (percentage && typeof percentage === 'string')
```

Reply:
done